### PR TITLE
buildly-core gitHub actions

### PR DESCRIPTION
**Pull request by abhishek-kumar-piyush**
## Purpose
Github actions to run unit test, create docker image and push it on GCP/AWS for dev, demo and prod branches.

### Further Info
Ticket number: #111 
 
@RadhikaPPatel

### CODE REVIEW

File: .github/workflows/unit_test.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name: Sensor Service Unit Test
```
Comment (RadhikaPPatel) [inline .github/workflows/unit_test.yml:1]: Correct this name to indicate core

**Review by RadhikaPPatel — CHANGES_REQUESTED** [review]
Please correct the name

File: .github/workflows/unit_test.yml (hunk 1 above)
Comment (abhishek-kumar-piyush) [reply .github/workflows/unit_test.yml]: done

**Review by RadhikaPPatel — APPROVED** [review]
Looks good

**Pull request merged**